### PR TITLE
FGTS-52 create level0 

### PR DESCRIPTION
**Pull request by fheysel**
Create MVP1 Level0. 
Add level0 tiles.
Add parallax background
Add tree and shrubbery background
Implement level layout
Add collectable star fragment 
Add spikes

### CODE REVIEW

File: godotproject/PlayerTongue.gd [hunk 1]
```diff
@@ -2,8 +2,8 @@ extends Node2D
 
 signal tongue_swing
 
-export (float) var shoot_speed = 600
-export (float) var max_shoot_dist = 250
+export (float) var shoot_speed = 100000
```
Comment (charasyn) [inline godotproject/PlayerTongue.gd]:
```suggestion
export (float) var shoot_speed = 1500
```
This should be a more reasonable shoot speed. To correct the project to use the shoot_speed in this file, you need to open Player.tscn in the editor, select the PlayerTongue scene, and hit the reset button beside the Shoot Speed in the inspector on the right. Can you make another commit into this branch with these changes? (feel free to try out other shoot speeds than 1500)

File: godotproject/PlayerTongue.gd [hunk 2]
```diff
@@ -70,8 +70,6 @@ func _process(delta):
 func _on_Player_tongue_start(facing):
 	if idle:
 		start_shoot(facing)
-	pass # Replace with function body.
-
```
Comment (charasyn) [inline godotproject/PlayerTongue.gd:74]: Are we ok to submit cleanups like this in otherwise unrelated changes? I think it's probably good bc that way you don't have to make a whole other change and PR for a tiny thing like this.

File: godotproject/Star.gd [hunk 3]
```diff
@@ -0,0 +1,6 @@
+extends Area2D
```
Comment (charasyn) [inline godotproject/Star.gd:1]: Hey can you break the Star into its own scene? If you go into Level0 in the editor, there should be an option when right-clicking the star's root node called "Save Branch To Scene". You might need to reset the Node2D -> Transform property in the newly created scene to 0.

**Review by charasyn — CHANGES_REQUESTED** [review]
Good work!!! There are a few changes that I think would be good to do now:
- making the Star into its own scene file so it's easier to place into the level
- fixing the shoot speed
Lmk what you think about these changes.
Again good work, this level is great!!

File: godotproject/PlayerTongue.gd (hunk 2 above)
Comment (fheysel) [reply godotproject/PlayerTongue.gd:74]: I agree, I think it should be good. I generally try to clean up small stuff like one liner deletions or old comments as I find them.

File: godotproject/PlayerTongue.gd (hunk 1 above)
Comment (fheysel) [reply godotproject/PlayerTongue.gd]: for sure!

File: godotproject/Star.gd (hunk 3 above)
Comment (fheysel) [reply godotproject/Star.gd:1]: Done! Nice catch

**Review by charasyn — APPROVED** [review]
Looks good!
